tests/graph.py

```diff
@@ -0,0 +1,37 @@
+from supporting_components.graph_io import *
+from algorithms.color_refinement import color_refinement
+
+
+def test_copy(graph):
+    """
+    Tests the copy method of the graph
+    :param graph: The graph to copy
+    :return: Whether or not the graph is copied correctly
+    """
+    copy = graph.copy()
+    return copy.is_equal(graph)
+
+
+files = ['colorref_smallexample_4_7', 'colorref_smallexample_6_15', 'colorref_smallexample_2_49']
+error_count = 0
+for file in files:
+    filename = '../test_graphs/color_refinement/' + file + '.grl'
+    with open(filename) as f:
+        L = load_graph(f, read_list=True)
```

Comment:
Ik heb in 1 van m'n PR's een input_output module waarin we met 1 methode files kunnen inlezen. Misschien dat we daar nadat alle PR's gemerged zijn hier even een refactor issue overheen gooien bijvoorbeeld. Zodat we dat overal kunnen toepassen, anders wordt alles nu dependent van elkaar.

Reply:
Zal er een issue voor aanmaken.